code/data-processing/assemble-historical-jhu.R

```diff
@@ -2,14 +2,15 @@ library(readr)
 library(purrr)
 library(tibble)
 library(here)
+library(covidData)
```

Comment:
I think we can't rely on the user having covidData installed already -- could be a new user, right?
```suggestion
```